ApplicationInsights/src/JavaScriptSDK/ApplicationInsights.ts

```diff
@@ -214,8 +215,8 @@ export class ApplicationInsights implements IAppInsights, ITelemetryPlugin, IApp
     /**
      * Logs that a page or other item was viewed.
      * @param IPageViewTelemetry The string you used as the name in startTrackPage. Defaults to the document title.
-     * @param customProperties Additional data used to filter events and metrics. Defaults to empty. If a user wants
-     *                         to provide a custom duration, it'll have to be in customProperties
+     * @param customProperties Additional data used to filter events and metrics. Defaults to empty. 
+     * If a user wants to provide duration for pageLoad, it'll have to be in pageView.customProperties.duration
```

Comment:
should this be `pageView.properties.duration`?

Reply:
yeah, did update it, not sure why its not showing up